gillespy2/solvers/cpp/variable_ssa_c_solver.py

```diff
@@ -1,320 +0,0 @@
-import gillespy2
```

Comment:
Removing this is great, but we should try to be backwards compatible, at least for now... The import for VariableSSACSolver can just map to the SSACSolver and throw a warning that this solver is deprecated...  if we change the default arg "variable" to True on the SSACSolver's init, then it will behave identically.